content/en/docs/started/installing-kubeflow.md

```diff
@@ -9,148 +9,242 @@ Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it prov
         <td>Charmed Kubeflow</td>
         <td>Canonical</td>
-        <td>Conformant Kubernetes</td>
-        <td>1.6</td>
-        <td><a href="/docs/distributions/charmed/">Docs</a></td>
-        <td><a href="https://charmed-kubeflow.io/">External Website</a></td>
+        <td>
+          All Conformant Kubernetes
+        </td>
+        <td>
+          <a href="https://charmed-kubeflow.io/">Website</a>
+        </td>
+        <td>
+          1.6.1
+        </td>
       </tr>
       <tr>
         <td>Kubeflow on Oracle Container Engine for Kubernetes</td>
         <td>Oracle</td>
-        <td>Oracle Cloud Infrastructure (OCI)</td>
-        <td>1.6</td>
-        <td>N/A</td>
-        <td><a href="https://github.com/oracle-devrel/kubeflow-oke">External Website</a></td>
+        <td>
+          Oracle Container Engine for Kubernetes (OKE)
+        </td>
+        <td>
+          <a href="https://github.com/oracle-devrel/kubeflow-oke">Website</a>
+        </td>
+        <td>
+          1.6.0
+        </td>
       </tr>
       <tr>
         <td>Kubeflow on vSphere</td>
         <td>VMware</td>
-        <td>vSphere</td>
-        <td>1.6</td>
-        <td>N/A</td>
-        <td><a href="https://vmware.github.io/ml-ops-platform-for-vsphere/">External Website</a></td>
+        <td>VMware vSphere</td>
+        <td>
+          <a href="https://vmware.github.io/ml-ops-platform-for-vsphere/">Website</a>
+        </td>
+        <td>
+          1.6.0
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+### Legacy Distributions
+
+The following table lists <b>legacy distributions</b> which have <b>not had a recent release</b> (within the last 6 months).
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Name</th>
+        <th>Maintainer</th>
+        <th>Target Platform</th>
+        <th>Link</th>
+        <th>Latest Release</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Argoflow</td>
+        <td>Argoflow Users</td>
+        <td>
+          All Conformant Kubernetes
+        </td>
+        <td>
+          <a href="https://github.com/argoflow">Website</a>
+        </td>
+        <td>
+          1.3.0
+        </td>
+      </tr>
+      <tr>
+        <td>Kubeflow on Azure</td>
+        <td>N/A <sup>(not maintained)</sup></td>
+        <td>
+          Azure Kubernetes Service (AKS)
+        </td>
+        <td>
+          <s>Website</s>
+          <br>
+          <sub><a href="/docs/distributions/azure/">Legacy Docs</a></sub>
+        </td>
+        <td>
+          1.2.0 <sup>[<a href="https://github.com/kubeflow/manifests/pull/1642">Release Notes</a>]</sup>
+        </td>
       </tr>
     </tbody>
   </table>
 </div>
 
 <a id="manifests"></a>
-## Install the Kubeflow Manifests manually
+<a id="install-the-kubeflow-manifests-manually"></a>
+## Raw Kubeflow Manifests
+
+{{% alert title="Warning" color="warning" %}}
+This method is for advanced users.
```

Comment:
I will disagree and commit with this statement since this PR does not change the wording.

Kubeflow should be easy to install even if not using a distribution and it also sets a low bar for the community provided manifests & installation instructions.

Reply:
@surajkota I think this is a discussion for another time. There has always been significant debate about what sort of support should be provided for users of the raw manifests.

However, as it currently stands, we do not offer any environment-specific support, and so we must include a disclaimer that this is for advanced users.

Reply:
> Please change the hard coded Kubeflow version to short code so we can update it when distribution is released
> 
> `{{% aws/latest-version %}}`
> 
> and update the release notes link to be generic - https://github.com/awslabs/kubeflow-manifests/releases

@surajkota I have incorporated your requeted changes in commit https://github.com/kubeflow/website/pull/3440/commits/dba927df670be0ac152c2237f9cae320d35fdc68